src/hdf5_time_series_storage.jl

```diff
@@ -115,6 +115,30 @@ function serialize_time_series!(
     return
 end
 
+function get_data_type(ts::TimeSeriesData)
+    if eltype(first(values(ts.data))) <: CONSTANT
+        return "CONSTANT"
+    elseif eltype(first(values(ts.data))) == POLYNOMIAL
+        return "POLYNOMIAL"
+    elseif eltype(first(values(ts.data))) == PWL
+        return "PWL"
+    else
+        error("$(eltype(first(values(ts.data)))) is not supported in forecast data")
+    end
+end
+
+function get_data_type(ts::T) where {T <: StaticTimeSeries}
```

Comment:
StaticTimeSeries is a subtype of TimeSeriesData, so this function should not be needed.